.gitignore

```diff
@@ -131,3 +131,7 @@ dmypy.json
 
 # Models
 *.onnx
+# include small test model for tests
+!tests/assets/onnx/model.onnx
```

Comment:
Would it make sense to have a small model on the Hub, e.g. under the `optimum` org?